### PR TITLE
EFRS-793 Fixed Warning/error message for face recognition methods und…

### PR DESCRIPTION
**Pull request by olehkhomyk**
Fixed Warning/error message for face recognition methods und…

### CODE REVIEW

File: ui/src/app/core/face-recognition/face-recognition.service.ts [hunk 1]
```diff
@@ -43,4 +47,54 @@ export class FaceRecognitionService {
     const formData = new FormData();
     return this.http.post(`${environment.userApiUrl}retrain`, formData, { headers: { 'x-api-key': model.apiKey }});
   }
+
+  private createUIRequest(url: string,  options = {}, params = {}): any {
+    const parsedParams = Object.keys(params).length ? `?${(new URLSearchParams(params)).toString()}` : '';
+
+    return {
+      'Request Headers': {
```
Comment (isobolev) [inline ui/src/app/core/face-recognition/face-recognition.service.ts:55]: why do we need these hardcoded request headers?

File: ui/src/app/core/face-recognition/face-recognition.service.ts [hunk 2]
```diff
@@ -43,4 +47,54 @@ export class FaceRecognitionService {
...
+        host: `${window.location.host}:${window.location.port}`,
+        'User-Agent': window.navigator.userAgent,
+        ...options
+      },
+      'Request Body': 'File...'
+    };
+  }
 }
+
+// Accept: application/json, text/plain, */*
```
Comment (isobolev) [inline ui/src/app/core/face-recognition/face-recognition.service.ts:72]: commented code; empty lines

File: ui/src/app/features/face-recognition/recognition-result/recognition-result.component.ts [hunk 3]
```diff
@@ -60,8 +60,10 @@ export class RecognitionResultComponent implements OnDestroy {
    */
   drawCanvas(box: any, face: any) {
     const img = new Image();
+    const resultFace = face.length > 0 ? face[0] : { face_name: 'Undefined', similarity: 0 };
```
Comment (isobolev) [inline ui/src/app/features/face-recognition/recognition-result/recognition-result.component.ts:63]: 'Undefined' as a string? if this is intentional it should come from translations

File: ui/src/app/core/face-recognition/face-recognition.service.ts (hunk 1 above)
Comment (olehkhomyk) [reply ui/src/app/core/face-recognition/face-recognition.service.ts:55]: To display it on UI. Now it renamed to Header and Added docs before this method to understand this logic.

**Review by isobolev — APPROVED** [review]

**Pull request merged**